src/main/java/object/Damageable.java

```diff
@@ -0,0 +1,10 @@
+package object;
+
+public interface Damageable {
+
+    void takeDamage(int damage);
+
+    boolean isBroken();
+
+    boolean isDamageable();
```

Comment:
interface Damageable 

then boolean isDamageable(); always returns true. no need for this method

Reply:
it isn't true, there could be `GameObject` that isn't Damageable. For example, there will be walls and different objections, bonusHolders also can't be damaged